python/taichi/lang/impl.py

```diff
@@ -226,34 +226,6 @@ def make_tensor_element_expr(_var, _indices, shape, stride):
                                           shape, stride))
 
 
-@taichi_scope
-def insert_expr_stmt_if_ti_func(ast_builder, func, *args, **kwargs):
```

Comment:
So, are we completely removing the experimental real function in this PR? (If so, I think the tests can be removed as well)

Reply:
No, I will remove it in another PR, this is only a buggy function that prevents using `print` in the real functions. I merged this function with ASTTransformer.build_Expr.